gqcpy/src/QCModel/CI/LinearExpansion_bindings.cpp

```diff
@@ -255,9 +250,16 @@ void bindLinearExpansions(py::module& module) {
          * MARK: Entropy
          */
 
+        .def(
+            "calculateShannonEntropy",
+            [](const LinearExpansion<double, SpinResolvedONVBasis>& linear_expansion) {
+                return linear_expansion.calculateShannonEntropy();
+            },
+            "Return the Shannon entropy (information content) of the wave function.")
+
```

Comment:
```suggestion
```
The `calculateShannonEntropy` API is part of the interface I believe, so you can delete this here.